File: mario.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
//#include <stlib.h>

int main(void)
{
    // Get input from user between 1 - 8
    int n = 0;
    do
    {
    n = get_int("Height: ");
    }
    while (n >= 9 || n <= 0);
    
    // Assign n to m for 'for' loop so it doesn' affect n
    int m = n;
    char space[128] = " ";
    char sp = ' ';
    
    for (int j = 1; j < n; j++)
    {
        strncat(space, &sp, 1);
    }
    
    for (int i = 0; i < n; i++)
    {
        m--;
        space[m] = '#';
        printf("%s\n", space);
    }
}

File: mario.py
# Get input from user between 1 and 8

while True:
    try:
        height = int(input("Height: "))
        if height < 1 or height > 8:
            False
        else:
            break
    except:
        print("Only numbers between 1 and 8 allowed")

for i in range(0, height, 1):
    for j in range(1,  height - i, 1):  # Print first space
        print(" ", end="")
    for k in range(0, i + 1, 1):        # Print fist hash
        print("#", end="")
    print("")